packages/features/webhooks/pages/webhooks-view.tsx

```diff
@@ -53,7 +42,11 @@ const WebhooksList = ({
   const hasTeams = profiles && profiles.length > 1;
 
   return (
-    <>
+    <SettingsHeader
+      title={t("webhooks")}
+      description={t("add_webhook_description", { appName: APP_NAME })}
+      CTA={webhooksByViewer.webhookGroups.length > 0 ? <CreateNewWebhookButton isAdmin={isAdmin} /> : null}
+      borderInShellHeader={false}>
```

Comment:
borderInShellHeader is always false in all three cases now: 1) skeleton 2) no webhook 3) webhooks exist

<img width="850" alt="Screenshot 2025-04-12 at 9 26 12 PM" src="https://github.com/user-attachments/assets/ad88d0a1-dff5-4d5c-a1e0-a06a4bd3e9e6" />
<img width="881" alt="Screenshot 2025-04-12 at 9 24 59 PM" src="https://github.com/user-attachments/assets/3a5a4d07-832b-47c7-9f07-d5ff7057d066" />
<img width="891" alt="Screenshot 2025-04-12 at 9 45 46 PM" src="https://github.com/user-attachments/assets/42d931e8-037b-4389-9a86-e2ed8fa3aaaa" />